src/expression_evaluator/function_evaluator.cpp

```diff
@@ -16,19 +15,13 @@ void FunctionExpressionEvaluator::init(const ResultSet& resultSet, MemoryManager
     if (expression->dataType.getLogicalTypeID() == LogicalTypeID::BOOL) {
         selectFunc = ((binder::ScalarFunctionExpression&)*expression).selectFunc;
     }
-    for (auto& child : children) {
-        parameters.push_back(child->resultVector);
-    }
 }
 
 void FunctionExpressionEvaluator::evaluate() {
     for (auto& child : children) {
         child->evaluate();
     }
-    if (execFunc != nullptr) {
```

Comment:
up to you, i think our compile/bind function can be a nullptr, better to make is consistent with these?